### PR TITLE
Update docs for Ingress-GCE related cluster addon

### PR DESCRIPTION
**Pull request by rramkumar1**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Update stale docs.

```release-note
None
```

/assign @bowei

### CODE REVIEW

File: cluster/addons/cluster-loadbalancing/glbc/README.md [hunk 1]
```diff
@@ -1,112 +1,9 @@
 # GCE Load-Balancer Controller (GLBC) Cluster Addon
 
-This cluster addon is composed of:
-* A [Google L7 LoadBalancer Controller](https://github.com/kubernetes/contrib/tree/master/ingress/controllers/gce)
-* A [404 default backend](https://github.com/kubernetes/contrib/tree/master/404-server) Service + RC
+This cluster addon is composed of a 404 default backend service and replication
+controller. On GCE, this 404 service is used in conjuction with the
```
Comment (MrHohn) [inline cluster/addons/cluster-loadbalancing/glbc/README.md]: nit: We had migrated from replication controller to deployment.

**Review by MrHohn — APPROVED** [review]
/lgtm

**Comment by MrHohn** [comment]
/hold
for the owner file nit

File: cluster/addons/cluster-loadbalancing/OWNERS [hunk 2]
```diff
@@ -1,10 +1,10 @@
-# See the OWNERS docs at https://go.k8s.io/owners
-
 approvers:
 - bowei
 - rramkumar1
 - mrhohn
+- mixia
```
[the remark this answers is not in the document]
Comment (rramkumar1) [reply cluster/addons/cluster-loadbalancing/OWNERS]: Done.

File: cluster/addons/cluster-loadbalancing/glbc/README.md (hunk 1 above)
Comment (rramkumar1) [reply cluster/addons/cluster-loadbalancing/glbc/README.md]: Done

File: cluster/addons/cluster-loadbalancing/glbc/README.md [hunk 3]
```diff
@@ -1,112 +1,9 @@
 # GCE Load-Balancer Controller (GLBC) Cluster Addon
 
-This cluster addon is composed of:
-* A [Google L7 LoadBalancer Controller](https://github.com/kubernetes/contrib/tree/master/ingress/controllers/gce)
-* A [404 default backend](https://github.com/kubernetes/contrib/tree/master/404-server) Service + RC
+This cluster addon is composed of a 404 default backend service and deployment.
+On GCE, this 404 service is used in conjuction with the
```
Comment (MrHohn) [inline cluster/addons/cluster-loadbalancing/glbc/README.md]: Turned out a typo failed the test :)
conjuction -> conjunction

Comment (rramkumar1) [reply cluster/addons/cluster-loadbalancing/glbc/README.md]: Done

**Pull request merged**